### PR TITLE
Allow inserting contacts again

### PR DESCRIPTION
**Pull request by kevindew**
Trello: https://trello.com/c/za4TTu3h/614-fix-issue-blocking-contact-inserts

See commits for details.

### CODE REVIEW

**Comment by benthorner** [comment]
I see what you mean about the test - it is a bit whacky, and not much concerned with the feature. Do you think it would be reasonable to move this into a new JavaScript spec for this component (wrapper)?

File: spec/support/accessible_autocomplete_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    # select the option
+    parent_node = autocomplete_input.find(:xpath, "..")
+    parent_node.first(".autocomplete__option").click
+
+    # wait until option is selected
+    within("##{for_id}-select", visible: false) do
+      if value
+        find(%{option[value="#{value}"]:checked}, visible: false)
+      else
+        find("option:checked", visible: false, text: option)
```
Comment (benthorner) [inline spec/support/accessible_autocomplete_helper.rb]: Looks good - not sure why we need this else clause right now, though?

Comment (kevindew) [reply spec/support/accessible_autocomplete_helper.rb]: Yeah - it isn't needed here. It was a bit of a pain to work out so put it in. I think we should use it in other places. I've pulled it out for now. Will try bring it back soon though before it slips my mind.

**Comment by kevindew** [comment]
@benthorner changed to data_module

File: app/views/components/docs/autocomplete.yml [hunk 2]
```diff
@@ -88,7 +88,7 @@ examples:
       name: autocomplete-with-hint-on-options
       label:
         text: Select your country
-      custom_template: true
+      js_module: autocomplete-with-hint-on-options
```
Comment (benthorner) [inline app/views/components/docs/autocomplete.yml]: Missed a bit...

Comment (kevindew) [reply app/views/components/docs/autocomplete.yml]: d'oh thanks!

**Pull request merged**